jetty-io/src/main/java/org/eclipse/jetty/io/ClientConnector.java

```diff
@@ -41,6 +44,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * <p>The client-side component that connects to server sockets.</p>
```

Comment:
@gregw I expanded the javadocs.
`@see` references are not possible because we can't really reference anything outside `jetty-io`.
The documentation is already plenty about how to use `ClientConnector`.